qdrant-landing/content/documentation/frameworks/memgpt.md

```diff
@@ -0,0 +1,40 @@
+---
+title: MemGPT
+weight: 2400
+---
+
+# MemGPT
+
+[MemGPT](https://memgpt.ai/) is a system that enables LLMs to manage their own memory and overcome limited context windows to
+
+- Create perpetual chatbots that learn about you and change their own personalities over time.
+- Create perpetual chatbots that can interface with large data stores.
+
+Qdrant is available as a storage backend in MemGPT for storing and semantically retrieving data.
+
+## Usage
+
+You can configure MemGPT to use either a Qdrant server or an in-memory instance via the `memgpt configure` command.
+
```

Comment:
General comment: While I used ` pip install -U pymemgpt` to set up the `memgpt` command, 

`memgpt configure` set up different prompts, which didn't match what you have here:

```
mike@Qdrant-Jang ~ $ memgpt configure
? Select LLM inference provider: (Use arrow keys)
 » openai
   azure
   local
```

I'm _guessing_ users need some other `memgpt` command to get to the list of storage options shown in your [MemGPT PR](https://github.com/cpacker/MemGPT/pull/1023/files#diff-ebba04859ec9cde65c68649dcd42eddc80428ca2d0de26d241bf0946210e6a30R555).

Reply:
You can see the storage options after selecting through these options.

Reply:
Currently you should see these options.

https://memgpt.readme.io/docs/storage